dao/src/test/resources/application-test.properties

```diff
@@ -26,6 +26,12 @@ cache.specs.sessions.maxSize=100000
 cache.specs.assets.timeToLiveInMinutes=1440
 cache.specs.assets.maxSize=100000
 
+cache.specs.customers.timeToLiveInMinutes=1440
+cache.specs.customers.maxSize=10000
+
+cache.specs.users.timeToLiveInMinutes=1440
+cache.specs.users.maxSize=10000
```

Comment:
other cache specs use TTL of 100000 (5 zeroes), customer cache specs uses TTL of 10000 (4 zeroes). Is this intentional?